qhub/render/__init__.py

```diff
@@ -191,3 +198,68 @@ def qhubignore_matches(_):
                     os.rmdir(root_path / dir)
                 except OSError:
                     pass  # Silently fail if 'saved' files are present so dir not empty
+
+
+def set_env_vars_in_config(config):
+    """
+
+    For values in the config starting with 'QHUB_SECRET_XXX' the environment
+    variables are searched for the pattern XXX and the config value is
+    modified. This enables setting secret values that should not be directly
+    stored in the config file.
+
+    NOTE: variables are most likely written to a file somewhere upon render. In
+    order to further reduce risk of exposure of any of these variables you might
+    consider preventing storage of the terraform render output.
+    """
+    private_entries = get_secret_config_entries(config)
+    for idx in private_entries:
+        set_qhub_secret(config,idx)
+
+
+def get_secret_config_entries(config,config_idx=None,private_entries=None):
```

Comment:
This function seems like a good candidate for some unit test, do you think you have time to write a few?

also, space after comma in this function too